operator/pkg/central/extensions/reconcile_tls.go

```diff
@@ -74,6 +75,20 @@ func (r *createCentralTLSExtensionRun) Execute(ctx context.Context) error {
 	if err := r.ReconcileSecret(ctx, "scanner-db-tls", scannerEnabled && !shouldDelete, r.validateScannerDBTLSData, r.generateScannerDBTLSData, true); err != nil {
 		return errors.Wrap(err, "reconciling scanner-db-tls secret")
 	}
+
+	if features.ScannerV4.Enabled() {
```

Comment:
Are operator changes in this PR strictly required to not break the operator? If not we should put them in another PR.

Reply:
I believe this was necessary to make the tests pass. I don't remember which test specifically, though.